libs/pmp-web/shared/util/src/lib/pipes/resolve-hours.pipe.spec.ts

```diff
@@ -1,8 +1,24 @@
 import { ResolveHoursPipe } from './resolve-hours.pipe';
 
 describe('ResolveHoursPipe', () => {
-  it('create an instance', () => {
-    const pipe = new ResolveHoursPipe();
-    expect(pipe).toBeTruthy();
+  let pipe: ResolveHoursPipe;
+
+  beforeEach(() => {
+    pipe = new ResolveHoursPipe();
+  });
+  it('expect(pipe.transform(1))', () => {
+    expect(pipe.transform(1)).toBe(`1 hour`);
+  });
+  it('expect(pipe.transform(24))', () => {
+    expect(pipe.transform(24)).toBe(`1 day`);
+  });
+  it('expect(pipe.transform(25))', () => {
+    expect(pipe.transform(25)).toBe(`1 day 1 hour`);
+  });
+  it('expect(pipe.transform(26))', () => {
+    expect(pipe.transform(26)).toBe(`1 day 2 hours`);
+  });
+  it('expect(pipe.transform(49))', () => {
+    expect(pipe.transform(49)).toBe(`2 days 1 hour`);
```

Comment:
Maybe leave only the biggest value. In this case 2 days.
And add also weeks, months and years.
Ask @ABartoszko for help. I think we had similar functionality in one of our projects ;)